<launch>
    <node name="voiceRecognition" pkg="xfyun_new" type="iat_publish" output="screen"/>
	<node name="simpleChat" pkg="xfyun_new" type="chat_subscribe" output="screen"/>
	<node name="soundplay_node" pkg="sound_play" type="soundplay_node.py"/>
	<node name="chatRequests" pkg="xfyun_new" type="pyrequests.py" output="screen" />
</launch>
	
    
